src/app/Library/Uploaders/Validation/ValidBackpackUpload.php

```diff
@@ -0,0 +1,160 @@
+<?php
```

Comment:
This class is SO FREAKING CLEAN.

![slow_clap_cap](https://user-images.githubusercontent.com/1032474/233600050-f89037d6-e8e4-4d5c-9547-6a0538deb706.gif)